financespy/memory_backend.py

```diff
@@ -0,0 +1,14 @@
+import collections
+
+class MemoryBackend:
```

Comment:
The prefix 'Backend' is very strange, what is the meaning of it?

Another point is your commits: they don't tell really anything about your mindset when decided to create this bunch of code!

https://github.com/RomuloOliveira/commit-messages-guide

Reply:
I was thinking on supporting multiple sources of input data: XLSX files, plain text files, relational databases, so I would end up with a XLSXBackend, FileSystemBackend, SQLBackend, and this is MemoryBackend - a source of input data directly from memory to facilitate testing.

The suffix "Backend" is not good indeed, it is a bit ambiguous. Do you have anything better in mind?